Clojure/src/chapter_1/chapter_1_q1.clj

```diff
@@ -1,33 +1,13 @@
 (ns chapter-1.chapter-1-q1)
 
-(defn- adjacencies? [str-seq]
-  "Given a string s, returns true if it contains any adjacencies, and false otherwise."
-  (loop [last-char nil
-         current-char (first str-seq)
-         remaining-chars str-seq]
-    (if (empty? remaining-chars)
-      false
-      (if (= current-char last-char)
-        true
-        (recur current-char
-               (first (rest remaining-chars))
-               (rest remaining-chars))))))
+(defn unique-coll-sort
+  "Find if collection is unique, search for repetitions in sorted string."
+  [coll]
+  (let [sorted-string (clojure.string/join (sort coll))]
+    (empty? (re-find #"(\w)\1+" sorted-string))))
```

Comment:
The `\w` in the regular expression means the sorting solution fails for strings that include repeated characters not in `[a-zA-Z_0-9]`. Example test case: `"###"`.